code/modules/mob/living/brain/brain_item.dm

```diff
@@ -118,7 +118,8 @@
 	if(brainmob) //if we aren't trying to heal the brain, pass the attack onto the brainmob.
 		O.attack(brainmob, user) //Oh noooeeeee
 
-	damaged_brain = TRUE //brain was attacked, they're pretty fragile.
+	if(O.force != 0 && !(O.item_flags & NOBLUDGEON))
```

Comment:
just `if(O.force && !(O.item_flags & NOBLUDGEON))`

Reply:
We actually support force < 0 weapons so this is correct.